tensorflow/contrib/tensorrt/README.md

```diff
@@ -1,8 +1,51 @@
-# Using TensorRT in TensorFlow
+# Using TensorRT in TensorFlow (TF-TRT)
 
-This module provides necessary bindings and introduces TRT_engine_op operator
-that wraps a subgraph in TensorRT. This is still a work in progress but should
-be useable with most common graphs.
+This module provides necessary bindings and introduces
+`TRTEngineOp` operator that wraps a subgraph in TensorRT.
+This module is under active development.
+
+## Installing TF-TRT
+
+Currently Tensorflow nightly builds include TF-TRT by default,
+which means you don't need to install TF-TRT separately.
+You can pull the latest TF containers from docker hub or
+install the latest TF pip package to get access to the latest TF-TRT.
+
+If you want to use TF-TRT on NVIDIA Jetson platform, you can find
+the download links for the relevant Tensorflow pip packages here:
+https://docs.nvidia.com/deeplearning/dgx/index.html#installing-frameworks-for-jetson
+
+## Installing TensorRT
+
+In order to make use of TF-TRT, you will need a local installation
+of TensorRT from the
+[NVIDIA Developer website](https://developer.nvidia.com/tensorrt).
+Installation instructions for compatibility with TensorFlow are provided on the
+[TensorFlow GPU support](https://www.tensorflow.org/install/gpu) guide.
```

Comment:
This portion is pretty similar to what it used to be, but it does raise a few questions, so clarifications could be useful. It used to reference `TensorRT 3.0.4`, which could be beneficial if an exact version is necessary. At the same time, the `NVIDIA Developer website` points you to `TensorRT 5` (is that supported yet?), and the `TensorFlow GPU support` page has a command to install `libnvinfer4=4.1.2-1+cuda9.0`.

I don't know where the definitive version should be documented, but right now it's a little unclear what is necessary as a user - If I try to follow these instructions directly, I'll end up with TensorRT 5 in addition to `libnvinfer4`, which doesn't seem like an ideal setup. If the `sudo apt install libnvinfer4=4.1.2-1+cuda9.0` gives me all the dependencies I need for this, I think linking to the `TensorFlow GPU support` as the primary resource would be the most intuitive, while the NVIDIA site could be supplementary.

Again, this PR doesn't change these instructions much, so this could be clarified outside of this PR, but as a user who has ran into confusion on this, some clarifications in the docs would be appreciated!

Reply:
I removed the NVIDIA link encouraging users to start from https://www.tensorflow.org/install/gpu for installing TensorRT to reduce the confusion about which TensorRT version to install. I hope that makes it a bit better. Thanks for pointing this out.